From 2deb91e40f38de897db24e2ea817cfd900bbf57a Mon Sep 17 00:00:00 2001
From: Zhipeng Gong <zhipeng.gong@intel.com>
Date: Wed, 4 May 2016 21:24:05 -0400
Subject: [PATCH 120/153] [VPG]: drm/i915: convert DRM_ERROR to DRM_DEBUG_DRIVER in hang check

There is no actual TDR reset type action being taken yet,
It is simply overzealous debug spew and not an actual issue.
---
 drivers/gpu/drm/i915/i915_irq.c |    9 +++------
 1 files changed, 3 insertions(+), 6 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_irq.c b/drivers/gpu/drm/i915/i915_irq.c
index 4ec7cea..9787c9c 100644
--- a/drivers/gpu/drm/i915/i915_irq.c
+++ b/drivers/gpu/drm/i915/i915_irq.c
@@ -3026,19 +3026,16 @@ static void i915_hangcheck_elapsed(struct work_struct *work)
 				ring->hangcheck.action = HANGCHECK_IDLE;
 
 				if (!list_empty(&ring->request_wait_list)) {
-					printk(KERN_ERR "<%s> \x1B[35mwait list active\x1B[0m\n", ring->name);
 					/* Do an explicit seqno check to catch stuck h/w. */
-//					/* Issue a wake-up to catch stuck h/w. */
+					/* Issue a wake-up to catch stuck h/w. */
 					if (!test_and_set_bit(ring->id, &dev_priv->gpu_error.missed_irq_rings)) {
 						if (!(dev_priv->gpu_error.test_irq_rings & intel_ring_flag(ring)))
-							DRM_ERROR("Hangcheck timer elapsed... %s idle\n",
+							DRM_DEBUG_DRIVER("Hangcheck timer elapsed... %s idle\n",
 								  ring->name);
 						else
-							DRM_INFO("Fake missed irq on %s\n",
+							DRM_DEBUG_DRIVER("Fake missed irq on %s\n",
 								 ring->name);
-						printk(KERN_ERR "<%s> \x1B[35mRing hung!\x1B[0m\n", ring->name);
 						i915_gem_request_notify(ring, false);
-						//wake_up_all(&ring->irq_queue);
 					}
 					/* Safeguard against driver failure */
 					ring->hangcheck.score += BUSY;
-- 
1.7.1

